tests/hubspot/test_hubspot.py

```diff
@@ -92,99 +78,17 @@ def test_hubspot_wrong_data_return(connector, datasource, mocker):
 
     req_mock.return_value.json = lambda: {'results': 'oui'}
 
-    try:
+    with pytest.raises(HubspotConnectorException):
         connector.get_df(datasource)
-    except Exception as e:
-        assert isinstance(e, HubspotConnectorException)
-
-    assert req_mock.called_once()
-
-
-def test_hubspot_get_webanalytics(connector, datasource, mocker):
```

Comment:
Is this test not important ?
It looks important for me... 🤔

Reply:
I have removed the part on webanalytics route
So this test is not necessary